Console/UpgradeHelperCommand.php

```diff
@@ -36,7 +36,11 @@ protected function execute(InputInterface $input, OutputInterface $output)
             $output->writeln('-------- ' . $type . ' --------');
             foreach ($items as $patched => $customized) {
                 $output->writeln('Patched: ' . $patched);
-                $output->writeln('Customized: ' . $customized);
+                if (is_array($customized)) {
+                    foreach ($customized as $customizedFiile) {
+                        $output->writeln('Customized: ' . $customizedFile);
```

Comment:
This can't work you has `$customizedFiile` in the loop but then refer to it as `$customizedFile`

Reply:
oops, missed 1/2 typo